extractor/docker/docker.go

```diff
@@ -73,7 +63,7 @@ type Extractor struct {
 	cache  cache.Cache
 }
 
-func NewDockerExtractor(option types.DockerOption, cache cache.Cache) (Extractor, error) {
+func NewDockerExtractor(option types.DockerOption) (Extractor, error) {
```

Comment:
You should keep `cache.Cache` the argument. If we remove it, `cache.Initialize` always take `utils.CacheDir`. 
https://github.com/aquasecurity/fanal/pull/67/files#diff-1ae53d6fc729587121b603c3ace40f24R78

But Trivy needs to specify the cache directory from a CLI option. Also, I want to avoid changing the interface.